tests/cli/test_report.py

```
@@ -55,6 +55,12 @@ def test_report():
     result.print_report()
 
 
+def test_report_detailed():
+    # buildtest report --detailed
+    Report(configuration=configuration, format_detailed=True)
+    Report(configuration=configuration, format_detailed=True, format_args="name")
+
```

this should raise an exception. Right you need to capture this 

```python
with pytest.raises(...):
  Report(...)

```